src/hotkey/command_executor.cpp

```diff
@@ -480,47 +477,47 @@ void basic_handler::handle_event(const SDL_Event& event)
 		// handled by the executor.
 		// If we're not in a dialog we can call the regular key event handler.
 		if (!gui::in_dialog()) {
-			key_event(*disp_, event.key,exec_);
+			key_event(*disp_, event,exec_);
 		} else if (exec_ != NULL) {
```

Comment:
Quite a lot of Null checks here,  why not break out completely in case exec_ is null?